ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -255,6 +258,34 @@ func (r *RayJobReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&rayv1alpha1.RayJob{}).
 		Owns(&rayv1alpha1.RayCluster{}).
 		Owns(&corev1.Service{}).
+		WithEventFilter(predicate.Funcs{
+			DeleteFunc: func(e event.DeleteEvent) bool {
+				r.Log.Info("event to delete", "event", e)
```

Comment:
let's make the `setupManager` cleaner and extract the logic here to a separate handler function?

Reply:
+1

Reply:
fixed